Worktez/worktez
# Fix/display errors

### Functionality:
Display Errors For all the Try Catch Blocks

### Solution:
Used getErrorCode Function to Get the errors and have set the showError to true.
Have included a nullable parameter to getErrorCode() function to access the apiError in ErrorCodes.xml file.

### Risk level:

### How to test:
Put `throw "myError";` in the try block and execute the containing function and check whether the Error Component shows the correct error code and Displays API_ERROR.
